File: z-clients/javascript/src/models/kvSetOut.ts
// this file is @generated

export interface KvSetOut {
    version: number;
}

export const KvSetOutSerializer = {
    // biome-ignore lint/suspicious/noExplicitAny: intentional any
    _fromJsonObject(object: any): KvSetOut {
        return {
            version: object['version'],
        };
    },

    // biome-ignore lint/suspicious/noExplicitAny: intentional any
    _toJsonObject(self: KvSetOut): any {
        return {
            'version': self.version,
        };
    }
}

File: z-clients/rust/src/models/kv_set_out.rs
// this file is @generated
use serde::{Deserialize, Serialize};

#[derive(Clone, Debug, Deserialize, Serialize)]
pub struct KvSetOut {
    pub version: u64,
}

impl KvSetOut {
    pub fn new(version: u64) -> Self {
        Self { version }
    }
}
